bids_prov/spm_parser.py

```diff
@@ -1,74 +1,102 @@
-import sys
-import click
+import argparse
 import json
 import os
 import re
-from difflib import SequenceMatcher
+import hashlib
 
 from collections import defaultdict
+from bids_prov import spm_load_config as conf
+from bids_prov import get_id
 
-from . import spm_load_config as conf
-from . import get_id
+def format_activity_name(activity_name: str, l_max=30) -> str:
+    """Function to limit name of activity
```

Comment:
Can you explicit which type of "limitation" is done? This is for me to better understand.

Reply:
previous considerations on activity name length that we didn't change when we start dev of spm_exporter. Exactly, we were thinking of asking you if we should change